pubspec.lock

```diff
@@ -1,369 +0,0 @@
-# Generated by pub
```

Comment:
probably don't want this file deleted

Reply:
maybe pop it in git ignore?

Reply:
I think I just removed it from git, but I can double check. It shouldn't be in version control